lab/lablib/SoundManager.js

```diff
@@ -31,6 +45,7 @@ export class SoundManager {
     await this.tone.start();
 
     this.init_synths();
```

Comment:
For now, change init_synths to gather up the instrument IDs from the scores and only create the synths as needed. In the future, I may want to customize this further.

Reply:
added this to my notes for now.